testsuite/gateway/gateway_api/gateway.py

```diff
@@ -56,6 +56,10 @@ def create_instance(cls, openshift: OpenShiftClient, name, hostname, labels, tls
     def service_name(self) -> str:
         return f"{self.name()}-istio"
 
+    def external_ip(self) -> str:
```

Comment:
Missing `@property` decorator, or?

Reply:
What about https? Wont the port be 443?

Reply:
Property is not there on purpose, I need to pass this function as function, which is difficult to do with property.

Regarding 443, You are right, that is an issue, the port should be changed based on what is in the listener, but given that TLS is always skipped right now on Kind, I will fix it in follow-up PR as it should not affect the tests if thats ok with you

Reply:
It can be fixed in another PR :+1: